zcu/encryption.py

```diff
@@ -20,27 +28,37 @@ def aes_decrypt(cipher, aes_key):
     encrypted_data = BytesIO()
     while True:
         aes_chunk = struct.unpack('>3I', cipher.read(12))
-        encrypted_data.write(cipher.read(aes_chunk[0]))
+        # aes_chunk[0] -> length not padded
+        # aes_chunk[1] -> padded length
+        if digi:
```

Comment:
```
chunk_size = aes_chunk[1] if digi else aes_chunk[0]
```

Reply:
I'm not a pythonista and I find the if...else...syntax clearer, but if you really feel it should be that way, I'll change it